wikiHandler.js

```diff
@@ -33,13 +35,14 @@ async function fetchAndFormatAuctionData(auction_user, auction_contents, server)
 
 function formatPriceMessage(item, data) {
     const price_points = [];
-    for (let interval in data[1]) {
-        price_points.push(`[*${interval}d*] ${data[1][interval]}`);
-    }
-    const price = data[0] !== undefined ? `**${data[0]}pp** ` : '';
-    return `${item} ${price}${price_points.join(" / ")}`;
```

Comment:
data[1] is an object so a just replaced your forEach with a for loop.

Reply:
heh that just undoes the change I was trying to do, sadly (get away from that type of for-loop). Maybe I can fix it differently...

Reply:
I guess it's just fine for now.